cmd/console/console.go

```diff
@@ -210,9 +210,10 @@ func modifySshdConfig() error {
 	return ioutil.WriteFile("/etc/ssh/sshd_config", []byte(sshdConfigString), 0644)
 }
 
+// https://www.freedesktop.org/software/systemd/man/os-release.html
 func writeOsRelease() error {
 	idLike := "busybox"
-	if osRelease, err := ioutil.ReadFile("/etc/os-release"); err == nil {
+	if osRelease, err := ioutil.ReadFile("/usr/lib/os-release"); err == nil {
```

Comment:
Doing it dynamically in `ros` simplifies console building. We should try to remove as many things as possible from the console building process so people can build their own more easily.